__tests__/app.test.js

```diff
@@ -0,0 +1,34 @@
+const request = require('supertest')
+const app = require('../app')
+const data = require('../db/data/test-data/index')
+const seed = require('../db/seeds/seed')
+
+beforeEach(() => {
+    return seed(data);
+})
+
+describe('GET Reviews', () => {
+    //console.log(reviewsData);
+    test('should return object with one entry: {"reviews" : <all reviews data>}', () => {
+        return request(app).get('/api/reviews')
+        .expect(200)
+        .then((res) => {
+            const reviews = res.body.reviews;
+
+            expect(Array.isArray(reviews)).toBe(true);
+            expect(reviews.length).toBe(13);
+            
+            reviews.forEach(review => {
```

Comment:
There are better jest matchers that can be utilised here, objectContaining, toMatchObject, toHaveProperty